src/roi/__tests__/computeRois.test.ts

```diff
@@ -32,7 +32,33 @@ test('test 2, waterShed for a grey image', () => {
   ]);
 
   const roiMapManager = waterShed(image, { threshold: 2 / 255 });
-  const rois = roiMapManager.getRois({ kind: 'bw' });
-  expect(rois[1].origin).toEqual({ column: 5, row: 5 });
-  expect(rois[0].origin).toEqual({ column: 2, row: 1 });
+
+  const result = new RoiMapManager({
+    data: getInt32Array(`
+           0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,
+          -1, -1, -1,  0,  0,  0,  0,  0,  0,  0, -1, -1,
+          -1, -1,  0,  0,  0,  0,  0,  0, -1, -1, -1, -1,
+           0,  0,  0,  0,  0,  0,  0,  0, -1,  0,  0,  0,
+           0,  0,  0,  0,  0,  0,  0,  0,  0, -2,  0,  0,
+           0,  0,  0,  0,  0,  0, -2, -2, -2,  0,  0,  0,
+           0,  0,  0,  0, -2, -2, -2, -2,  0,  0,  0,  0,
+           0, -2, -2, -2, -2,  0,  0,  0,  0,  0,  0,  0,
+           0,  0, -2,  0
+        `),
```

Comment:
It is really hard to try to understand this test when it written out with a number of columns which is different from the number of columns of the image the result is based on.

Reply:
![CleanShot 2023-12-14 at 10 47 07](https://github.com/image-js/image-js-typescript/assets/101188881/a3b2185e-bfb3-4467-93de-be9ddf421bcf)

I have a question regarding this test.  I check ROIs for their origins but the second ROI starts before actual ROI points(column: 5 row: 5). It is due to the fact that computeRois checks minimum column and minimum row separately. In this case the origin forms a point that doesn't technicall belong to ROI. My question is this an acceptable behavior?

![CleanShot 2023-12-14 at 10 49 44](https://github.com/image-js/image-js-typescript/assets/101188881/00321775-a2bc-4112-9d04-01f52c38fa41)

Reply:
That is normal. I asked Luc and he also thinks it is normal and expected.

The origin does not have to be a point of the ROI. Maybe it's just a terminology issue, but I cannot think of a better term for this.